### PR TITLE
Feat/300 about us styling fixx

### LINKED ISSUE #300: style(about-us): improve about us styling
### Suggestions from Dec 5 meeting

- [ ] Align description to picture
- [ ] Make description bigger
- [ ] Bold "We Are The Red Pandas"
- [ ] Shrink white border around the entire page
- [ ] Make Our Story and Our Team boxes smaller

### PR DESCRIPTION
**Pull request by z1tang**
align description to picture
add team picture
bold "we are the red pandas"
shrink white border
make boxes small
in case for the description to flow outside the box when shrink the window, I used  "overflow: scroll; " 
closes #300

### CODE REVIEW

File: source/components/about-us/about-us.css [hunk 1]
```diff
@@ -3,54 +3,51 @@ main {
...
-  height: 1000px;
-  margin: 100px;
+  height: 800px;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
 }
 #introduction-title-grid {
-  height: 400px;
+  height: 200px;
```
Comment (edbertdai) [inline source/components/about-us/about-us.css:12]: I think the title should be centered within the blue.
Replace with this CSS to fix it:
```suggestion
  line-height: 200px;
```

**Review by edbertdai — COMMENTED** [review]
Apply the changes I suggested and I think it looks good

File: source/components/about-us/about-us.css [hunk 2]
```diff
@@ -3,54 +3,51 @@ main {
...
+  margin-right: 4vw;
+  margin-bottom: 100px;
+  font-size: 150%;
   text-align: left;
+  line-height: 150%;
 }
 #our-team {
-  margin: 100px 100px 50px;
-  height: 400px;
+  height: 200px;
```
Comment (edbertdai) [inline source/components/about-us/about-us.css:44]: Same as introduction title above.
```suggestion
  line-height: 200px;
```

**Comment by jbutera94** [comment]
Once Edbert's suggestions are addressed, looks good to me

**Review by edbertdai — APPROVED** [review]
Thanks, looks pretty nice now

**Pull request merged**